src/app/modules/app/pages/users/users.component.ts

```diff
@@ -24,16 +29,19 @@ export class UsersComponent {
 
   public isSidenavOpen = true;
   public isLoading = true;
-
+  userRoles: string[] | undefined;
```

Comment:
Can we have more than 1 role per user?

Reply:
If we have more than 1 role per user, we will have 2 sidebars and tables

Reply:
Keycloak realm already returns the token with a roles array, containing at least 3 other realms roles in the token besides trainer or admin

Reply:
> If we have more than 1 role per user, we will have 2 sidebars and tables
The token already has more than one role per user and there is only one sidebar and one table

Reply:
If a user has the admin and trainer roles, the 2 tables and sidebars will be shown
https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/blob/a3c2e746fe17a75722542df3d959e17ed6b1c6f8/src/app/modules/app/components/sidebar/sidebar.component.html#L11

https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/blob/a3c2e746fe17a75722542df3d959e17ed6b1c6f8/src/app/modules/app/components/sidebar/sidebar.component.html#L23

Reply:
As I understand it, for the moment there is no way to create a user with both admin and trainer roles without manually assigning the other role on the keycloak server. Every user should have only one of the two roles.

Reply:
Cool :)